packages/kbn-unified-field-list/src/components/field_popover/field_popover.tsx

```diff
@@ -54,10 +70,15 @@ export const FieldPopover: React.FC<FieldPopoverProps> = ({
       {...otherPopoverProps}
     >
       {isOpen && (
-        <>
-          {content && header ? <EuiPopoverTitle>{header}</EuiPopoverTitle> : header}
-          {content}
-        </>
+        <EuiFlexGroup gutterSize="none" direction="column" css={{ maxHeight: '90vh' }}>
+          {Boolean(header) && (
+            <EuiFlexItem grow={false}>
+              {content ? <EuiPopoverTitle>{header}</EuiPopoverTitle> : header}
+            </EuiFlexItem>
+          )}
+          {content ? <EuiFlexItem css={{ overflowY: 'auto' }}>{content}</EuiFlexItem> : content}
```

Comment:
There seems to be a small issue with the scroll indicator being visible during stats loading:

![Oct-07-2024 10-19-12](https://github.com/user-attachments/assets/ef62d641-d0f1-4088-890f-2d8aa9f8c1d7)

Does it need a min height?

Reply:
Good catch! I fixed it with padding and negative margins, and also added EUI scroll styles: https://github.com/elastic/kibana/pull/195147/commits/f63fe9cef87099911552a29ccdcfbe90bfbce073.